### PR TITLE
[PAR-PT] Merwan & Justine

### PR DESCRIPTION
**Pull request by merwanguerrib**
Continuing what we've started with @justinezahiri

@ta-web-paris

### CODE REVIEW

File: starter-code/pizza.js [hunk 1]
```diff
@@ -1 +1,94 @@
...
+   var pizzaPrice = basis + ingredientsPrice;
+   $("aside strong").html(pizzaPrice);
+  console.log(pizzaPrice);
+ };
+  // Select the toppings with the 3 buttons
+  $(".btn-pepperonni").click(function() {
+    $(".btn-pepperonni").toggleClass("active");
+    $(".pep").toggle();
+    $("ul > li:contains(pepperonni)").toggle();
+    CalculatePrice(pizzaPrice)
```
Comment (ta-webpt-par) [inline starter-code/pizza.js:42]: The `CalculatePrice` function doesn't take any arguments, so here the parameter `pizzaPrice` is ignored

File: starter-code/pizza.js [hunk 2]
```diff
@@ -1 +1,94 @@
...
+var pizzaPrice = basis + ingredientsPrice;
+
+// Updating the price of the pizza with the right ingredients
+console.log(pizzaPrice);
+ // Callback function to calculate the price
+ function CalculatePrice(){
+  if ($("ul > li:contains(pepperonni)").is(":visible")){
+    ingredientsPrice = ingredientsPrice + pepPrice;
+  }
+  else if ($("ul > li:contains(mushrooms)").is(":visible")){
```
Comment (ta-webpt-par) [inline starter-code/pizza.js:21]: You shouldn't use `else if` here, if you have pepperoni on your pizza, every time you call this function the price will be increased by 1

**Comment by ta-webpt-par** [comment]
Good work ! Try to finish it when you have some time

File: starter-code/pizza.js (hunk 2 above)
Comment (merwanguerrib) [reply starter-code/pizza.js:21]: I couldn't figure it out another way, that's why I sent a message on slack yesterday to have a hint on what should I start with...

**Pull request closed without merging**